cursorless-talon/src/modifiers/containing_scope.py

```diff
@@ -43,6 +43,15 @@
     "tags": "xmlBothTags",
     "start tag": "xmlStartTag",
     "end tag": "xmlEndTag",
+    # LaTeX
+    "command": "latexCommand",
+    "environment": "latexEnvironment",
+    "part": "latexPart",
+    "chapter": "latexChapter",
+    "subsection": "latexSubsection",
+    "subsubsection": "latexSubSubSection",
+    "paragraph": "latexParagraph",
+    "subparagraph": "latexSubParagraph",
```

Comment:
I would be inclined to remove the `latex` prefix from all of these.  We have found that it can be helpful to be a bit less verbose for these scope types so that they can be used elsewhere